### PR TITLE
feat(acceptance): Add Statistics about what feature was omitted when accepting a feature

### PR DESCRIPTION
**Pull request by kelvin-lu**
If a feature succeeded the acceptance phase, locate and find metadata about what other feature had to be omitted to allow this new feature to be accepted.

Apologies for the huge # of lint changes, my linter has gotten a lot more aggressive in the past year. I can remove the ones around single/double quotes, but I think the addition of semicolons in some parts is overall good.

### CODE REVIEW

**Comment by micahjsmith** [comment]
Thanks this is a super useful feature! Sorry for the delay in reviewing I was offline last week

No worries about the lint changes :) maybe you can update package.json lint script? then please go ahead and run it on master, commit, and push to master. this will minimize the diff on this PR.

File: index.js [hunk 1]
```diff
@@ -1,186 +1,220 @@
...
+      const omittedFeatureMessage = omittedFeature
+        ? `We found that your feature provided more information than another feature: ${omittedFeature}`
+        : `We found that your feature added valuable information on top of all the existing features`;
       const message = dedent`
-        After validation, your feature was accepted. It will be automatically merged into the project.
-      `
-      await github.commentOnPullRequest(context, travisBuild.pull_request_number, message)
-      await github.mergePullRequest(context, travisBuild.pull_request_number)
-      await github.closePullRequest(context, travisBuild.pull_request_number)
+        After validation, your feature was accepted.
```
Comment (micahjsmith) [inline index.js:79]: Can this be a single paragraph with no newlines? It will render better on GitHub.

**Comment by kelvin-lu** [comment]
@micahjsmith I've updated the PR and master to use eslint - so the diff here should be minified!

**Review by micahjsmith — APPROVED** [review]

**Comment by micahjsmith** [comment]
Thanks @kelvin-lu!

**Pull request merged**